crates/datafusion_planner/src/relation/mod.rs

```diff
@@ -133,9 +135,9 @@ impl<'a, S: AsyncContextProvider> SqlQueryPlanner<'a, S> {
         }
     }
 
-    /// Get a constant expression literal from a function argument.
-    ///
-    /// Returns an optional name for the argument.
+    // Get a constant expression literal from a function argument.
+    //
+    // Returns an optional name for the argument.
```

Comment:
Just curious, is there a reason you made this a normal comment instead of a doc-comment?